amundsen_application/static/js/components/TableDetail/WatermarkLabel/index.tsx

```diff
@@ -10,27 +10,44 @@ interface WatermarkLabelProps {
 }
 
 class WatermarkLabel extends React.Component<WatermarkLabelProps> {
-
   constructor(props) {
     super(props);
-    this.getWatermarksLabel = this.getWatermarksLabel.bind(this);
   }
 
   render() {
     return (
-        <div className="watermark-label">{this.getWatermarksLabel(this.props.watermarks)}</div>
-      )
+      <div className="watermark-label">
+        <img className="range-icon" src="/static/images/watermark-range.png"/>
+        { this.getWatermarkText() }
+      </div>
+    );
   }
 
-  getWatermarksLabel(watermarks: Watermark[]) {
-    const low = watermarks.find((wtm) => wtm.watermark_type === "low_watermark");
-    const high = watermarks.find((wtm) => wtm.watermark_type === "high_watermark");
+  getWatermarkText = () => {
```

Comment:
nit: This method is returning markup and not just some text. Could be more appropriately named `renderSomethingSomething()`